samples/dashboard/demos/dashboard-component-kpi/demo.js

```diff
@@ -0,0 +1,155 @@
+    }, {
+        cell: 'kpi-10',
+        type: 'kpi',
+        title: 'Change',
+        value: 222,
+        valueFormatter: v => `${(v / 10).toFixed(1)}%`
+    }, {
+        cell: 'kpi-11',
+        type: 'kpi',
+        title: 'Cash',
+        value: 88,
+        valueFormat: '${value:,.2f}',
+        subtitle: {
+            type: 'diffpercent'
+        }
+    }, {
+        cell: 'kpi-12',
+        type: 'kpi',
+        title: 'Progress',
+        chartOptions: {
+            chart: {
+                type: 'solidgauge'
+            },
+            pane: {
+                startAngle: -150,
+                endAngle: 150,
+                background: {
+                    backgroundColor: '#EEE',
+                    innerRadius: '60%',
+                    outerRadius: '100%',
+                    shape: 'arc'
+                }
+            },
+            yAxis: {
+                min: 0,
+                max: 1000
+            },
+            series: [{
+                dataLabels: {
+                    enabled: false
+                },
+                rounded: true,
+                data: [{
+                    color: Highcharts.getOptions().colors[0],
+                    y: 70,
+                    innerRadius: '60%',
+                    outerRadius: '100%'
+                }]
+            }]
+        }
+    }, {
+        cell: 'kpi-13',
+        type: 'kpi',
+        title: 'Visits last 24 hours',
+        chartOptions: {
+            yAxis: {
+                min: 0,
+                max: 1000,
+                visible: true
+            },
+            series: [{
+                data: [130, 405, 200, 500, 100, 300, 200, 150],
+                clip: false
+            }]
+        }
+    }],
+    gui: {
+        enabled: true,
+        layouts: [{
+            id: 'layout-1',
+            rows: [{
+                cells: [{
+                    id: 'kpi-00'
+                }, {
+                    id: 'kpi-01'
+                }, {
+                    id: 'kpi-02'
+                }, {
+                    id: 'kpi-03'
+                }]
+            }, {
+                cells: [{
+                    id: 'kpi-10'
+                }, {
+                    id: 'kpi-11'
+                }, {
+                    id: 'kpi-12'
+                }, {
+                    id: 'kpi-13'
+                }]
+            }]
+        }]
+    }
+});
+
+function setValues(element) {
+    const chart = element.component.chart,
+        randomValue = random(1000);
+
+    if (chart && chart.options.chart.type !== 'solidgauge') {
+        chart.series[0].addPoint(
+            randomValue,
+            true,
+            true
+        );
+    } else if (chart) {
+        chart.series[0].setData([randomValue]);
+    }
+    element.component.update({
+        value: randomValue
+    });
+}
+
+dashboard.mountedComponents.forEach(element => {
+    setValues(element);
+    setInterval(() => {
+        setValues(element);
+    }, 1000);
```

Comment:
I'm not convinced to it. Maybe we should run a single setInterval and run a loop inside the function?